features/visitor_can_read_a_specific_article.feature

```diff
@@ -5,11 +5,20 @@ Feature: Visitor can read a specific article
     I would like to be able to click on an article and have it displayed
 
     Background: 
+        Given the following categories exist
```

Comment:
No need to create categories like this, take a look at the step definition for creating the articles

Reply:
Will you please clarify for us... we thought this is how it is done in the feature file?

Reply:
done